rotkehlchen/externalapis/etherscan.py

```diff
@@ -187,12 +188,9 @@ def _query(
         api_key = self._get_api_key()
         if api_key is None:
             if not self.warning_given:
-                self.msg_aggregator.add_warning(
-                    f'You do not have an {self.chain} Etherscan API key configured. rotki '
-                    f'etherscan queries will still work but will be very slow. '
-                    f'If you are not using your own ethereum node, it is recommended '
-                    f'to go to https://{self.base_url}/register, create an API '
-                    f'key and then input it in the external service credentials setting of rotki',
+                self.msg_aggregator.add_message(
+                    message_type=WSMessageType.MISSING_API_KEY,
+                    data={'service': 'etherscan', 'blockchain': self.chain.serialize()},
```

Comment:
not 100% sure if is fine to have `etherscan` hardcoded here or is better to use `ExternalLocation.ETHERSCAN`  along the blockcahin field. I would do that since in the other places like beacon chain and opensea I believe is good to use the same constant. What do you think?

Reply:
Can be like that, yes